osdconfig/proto/config.proto

```diff
@@ -0,0 +1,120 @@
+syntax = "proto3";
+
+// this package defines the configuration parameters and interfaces
+// requires to run portwork services
+package proto;
+
+service Spec {
+    rpc GetGlobalSpec(Empty) returns (GlobalConfig) {}
+    rpc SetGlobalSpec(GlobalConfig) returns (Ack) {}
+    rpc GetClusterSpec(Empty) returns (ClusterConfig) {};
+    rpc SetClusterSpec(ClusterConfig) returns (Ack) {};
+    rpc GetNodeSpec(NodeID) returns (NodeConfig) {};
+    rpc SetNodeSpec(NodeConfig) returns (Ack) {};
+}
+
+message Empty {}
+message Ack {
+    int64 n = 1;
```

Comment:
nit: Should we rename 'n' to something more readable or have error/return codes for the ack responses. Just suggesting, I am fine with what the mass decides.

Reply:
i got 'n' from typical Go's write() methods that return number of bytes written. Yes, some description on 'n' would help as it is not evident by itself as to what it does. Also okay to changing it to something else. Let's discuss today.